### ISSUE TITLE
Add Options to Email Subscriptions & Listservs

### ISSUE THREAD

**Issue by peterfiala** [enove/NoSLB#115]
We need to add the following to Email Subscriptions
- Subscribe to your regional emaillist (tied to the zip code in the person's account)
- Training Updates
- Members of the Press

If we could add the following to listservs:
- Subscribe to your regional listserv
- One question about listservs. We have several that are closed to the public. Will only the users who are on the list be able to manage their subscriptions?

**Comment by micchickenburger** [comment]
Are `Training Updates` and `Members of the Press` Listservs?

No problem with adding regional listserv functionality.

There's no great way to already show listservs that people have previously signed up for, because we haven't really linked their accounts.  We may have to make the modification manually in the future.  You will likely receive questions from people who report the website shows them as having not signed up, even though they already have been.  They could just flip the switch under their profile in this case, which would really do nothing.

So basically, no, I can't show private Listservs to people who are already subscribed to them because the website can't query members of a Listserv.

**Comment by peterfiala** [comment]
Thanks. I wonder if we could add a disclaimer that lets people know this? Something like:

If you are already subscribed for a WCASA listserv, that information will not display here. In order for your profile to accurately reflect your memberships, please select the listservs you are interested in. You will still only be subscribed once. Thank you.

**Comment by eoghanTadhg** [comment]
Ideally Vertical Response would be updated as well as Listserv list when toggling from subscriptions panel.
